src/components/Navbar.js

```diff
@@ -1,10 +1,13 @@
 import './Navbar.css';
+import { useNavigate } from "react-router-dom";
 
 function Navbar() {
+  let navigate = useNavigate();
+
   return (
     <nav>
       <a className='company-logo-navbar' href="/">OWOW</a>
-      <a className='login-text-navbar' href="/">Login</a>
+      <a className='login-text-navbar' onClick={() => navigate("/login")}>Login</a>
```

Comment:
You could make this button a separate component that could change based on the authentication state.
Login if token is null and signout if token is available for example